tests/framework/randomness/test_index_map.py

```diff
@@ -38,7 +49,7 @@ def generate_keys(number, types=("int", "float", "datetime"), seed=123456):
             keys["int"] = np.arange(start, start + number, dtype=int)
 
     if "float" in types:
-        keys["float"] = rs.random_sample(size=number)
+        keys["float"] = rs.random_sample(number)
```

Comment:
what drove removing the keyword?

Reply:
Not sure why I had deleted this but it is back in there.